### PR TITLE
add stop_factor_lr in FactorScheduler

### PR DESCRIPTION
**Pull request by tornadomeet**

### CODE REVIEW

File: python/mxnet/lr_scheduler.py [hunk 1]
```diff
@@ -71,8 +72,11 @@ def __call__(self, num_update):
         if num_update > self.count + self.step:
             self.count += self.step
             self.base_lr *= self.factor
-            logging.info("Update[%d]: Change learning rate to %0.5e",
-                         num_update, self.base_lr)
+            if self.base_lr < self.stop_factor_lr:
+                self.base_lr = self.stop_factor_lr
```
Comment (piiswrong) [inline python/mxnet/lr_scheduler.py]: This changes default behavior. Please add a warning here

**Comment by tornadomeet** [comment]
@piiswrong Done!  but the travis fail with uncorrelated errors accidental(i met this two times today), you may have check that.

**Comment by mli** [comment]
cpptest fails randomly. we'd better remove it to avoid confusing people.

**Comment by piiswrong** [comment]
@mli Can we fix it?

**Comment by tornadomeet** [comment]
@piiswrong @mli  can you review again this pr?

**Pull request merged**